packages/shadcn/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@agents-ui",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "index.ts",
+  "scripts": {
+    "registry:build": "shadcn build --output ./dist",
+    "registry:serve": "python3 -m http.server 3210 -d dist"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
question: Is this license correct? I don't think it matches with [the license at the root of the repository](https://github.com/livekit/components-js/blob/main/LICENSE).